lib/msteams_hermes/components/adaptive_card.rb

```diff
@@ -23,6 +23,8 @@ def initialize(body:, actions: nil, entities: [])
 
         @entities = entities
         raise "AdaptiveCard `entities` must be an Array" unless @entities.nil? || @entities.is_a?(Array)
+
+        set_width(full_width: full_width)
```

Comment:
If you keep this method you should be able to omit the variable:
```suggestion
        set_width(full_width:)
```

Reply:
The prefix `set` appears to me somewhat of a code smell, because this `getter/setter` convention is not ruby-ish, rather Java-ish